### PR TITLE
Updated Installation.md by adding proper documentation regarding setting up the talawa app

### PR DESCRIPTION
**Pull request by AntoTomAbraham**
Solved issue #1384

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -43,13 +43,22 @@ We have tried to make the process simple. Here's what you need to do.
...
+You need to setup your own local instance of [Talawa-API](https://github.com/PalisadoesFoundation/talawa-api) and [Talawa-admin](https://github.com/PalisadoesFoundation/talawa-admin). The advantage is that you'll be working with the latest code.
 
-1. If you have set up all the 3 projects locally ([talawa](https://github.com/PalisadoesFoundation/talawa), [talawa-api](https://github.com/PalisadoesFoundation/talawa-api), and [talawa-admin](https://github.com/PalisadoesFoundation/talawa-admin)), then use the URL (Organization URL) in this form:
-    1. http://(localhost IP):4000/graphql (See the next item for the IP addresses to use for your environment)
-1. Every Emulator has a different localhost IP address. 
-    1. For the Android Studio emulator use: http://10.0.2.2:4000/graphql
+1. You need to setup all the 3 projects locally ([talawa](https://github.com/PalisadoesFoundation/talawa/blob/develop/INSTALLATION.md), [talawa-api](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md), and [talawa-admin](https://github.com/PalisadoesFoundation/talawa-admin/blob/develop/INSTALLATION.md)).
```
Comment (palisadoes) [inline INSTALLATION.md:52]: Please find a way to exclude links that include a specific branch. If the default branch changes, then the documentation will be incorrect.

File: INSTALLATION.md [hunk 2]
```diff
@@ -43,13 +43,22 @@ We have tried to make the process simple. Here's what you need to do.
...
+2. After setting up [talawa-api](https://github.com/PalisadoesFoundation/talawa-api/blob/develop/INSTALLATION.md) and [talawa-admin](https://github.com/PalisadoesFoundation/talawa-admin/blob/develop/INSTALLATION.md) create an account in talawa admin.
```
Comment (palisadoes) [inline INSTALLATION.md:53]: Please find a way to exclude links that include a specific branch. If the default branch changes, then the documentation will be incorrect.

Comment (palisadoes) [inline INSTALLATION.md:53]: What type of account and why?

File: INSTALLATION.md [hunk 3]
```diff
@@ -43,13 +43,22 @@ We have tried to make the process simple. Here's what you need to do.
...
+3. After creating an account in talawa admin edit data in mongodb document under user collection by changing userType from ADMIN to SUPERADMIN and adminApproved from false to true
```
Comment (palisadoes) [inline INSTALLATION.md:54]: Why isn't this possible when creating an account? Do we need to update Talawa-Admin to allow this?

File: INSTALLATION.md [hunk 4]
```diff
@@ -43,13 +43,22 @@ We have tried to make the process simple. Here's what you need to do.
...
+
+<p align="center"><img width="946" alt="createO" src="https://user-images.githubusercontent.com/64683098/212376827-970ce7ca-d8fa-419b-ba90-691c93a25649.png"></p>
+<p align="center"><img width="946" alt="createO" src="https://user-images.githubusercontent.com/64683098/212393122-d1715819-31d0-4dd0-b072-9bb24622f8e5.png"></p>
+
+4. After creating account, login and create an organisation in talawa-admin
```
Comment (palisadoes) [inline INSTALLATION.md:59]: Please be more specific, which account?

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please make the requested updates. Refer to any uploaded images more specifically. We really need to have this documentation done correctly.

**Comment by AntoTomAbraham** [comment]
I have made the necessary changes, please review the PR

**Review by palisadoes — APPROVED** [review]

**Pull request merged**